src/main/resources/values/values/strings.xml

```diff
@@ -79,37 +79,37 @@
     <string name="settings_download">Download</string>
     <string name="settings_general">General</string>
     <string name="settings_library">Library</string>
-    <string name="settings_parental_controls">Parental Controls</string>
+    <string name="settings_parental_controls">Parental controls</string>
     <string name="settings_reader">Reader</string>
     <string name="settings_security">Security</string>
     <string name="settings_server">Server</string>
     <string name="settings_tracking">Tracking</string>
 
     <!-- Settings Menu Titles-->
-    <string name="settings_advanced_screen">Advanced Settings</string>
-    <string name="settings_appearance_screen">Appearance Settings</string>
-    <string name="settings_backup_screen">Backup Settings</string>
-    <string name="settings_browse_screen">Browse Settings</string>
-    <string name="settings_download_screen">Download Settings</string>
-    <string name="settings_general_screen">General Settings</string>
-    <string name="settings_library_screen">Library Settings</string>
-    <string name="settings_parental_control_screen">Parental Control Settings</string>
-    <string name="settings_reader_screen">Reader Settings</string>
-    <string name="settings_security_screen">Security Settings</string>
-    <string name="settings_server_screen">Server Settings</string>
-    <string name="settings_tracking_screen">Tracking Settings</string>
+    <string name="settings_advanced_screen">Advanced</string>
```

Comment:
If it said "Settings" above all these, they could be combined with other strings.

Reply:
These strings are shown at the top of the relevant settings menu, there isn't really a good way to combine them

Reply:
I was thinking in terms of Android-strings, so that they don't have to be translated twice.
There is a setting to mitigate it in Weblate though, which propagates existing translations across the component.